src/server.c

```diff
@@ -3404,14 +3424,16 @@ int processCommand(client *c) {
     if ((c->flags & CLIENT_PUBSUB && c->resp == 2) &&
         c->cmd->proc != pingCommand &&
         c->cmd->proc != subscribeCommand &&
+            c->cmd->proc != ssubscribeCommand &&
         c->cmd->proc != unsubscribeCommand &&
+            c->cmd->proc != sunsubscribeCommand &&
```

Comment:
indentation

Reply:
Damn. method rename by the IDE caused it. :|